dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/ws/connectionpool/monitor/ConnectionPoolMonitor.java

```diff
@@ -133,6 +133,7 @@ public void incCreateCount(@This Object fpObject) {
             cStats.incCreateCount();
             cStats.incManagedConnectionCount();
             cStats.incFreeConnectionCount();
+            cStats.updateWaitTime(0);
```

Comment:
This seems a little off for two reasons:
1. I think we could end up here after waiting (which would result in an extra 0 being added to the count in addition to the time we waited).
2. This would only include creating new connections.  I would expect that a 0 would also be added for when a connection is obtained from the free pool.